.travis.yml

```diff
@@ -21,7 +21,7 @@ install:
   - conda install conda-build
   - conda update -q conda conda-build
   - conda config --append channels conda-forge
-  - conda config --append channels skywalker-tag
+  - conda config --append channels skywalker-dev
```

Comment:
I'd rather we add a `happi` release than rely on the dev `happi` channel

Reply:
I kind of disagree. Seems like getting the full feature set for this plugin might take a few iterations in each library. I realize version numbers are cheap, but I don't think making a large number of pedantic versions is the way forward.

I'll yield for now.

Reply:
I see. I guess I made the assumption that `happi` was at a release point if we were ready to pull it into what will become a versioned `hutch-python` by the end of the month.

Reply:
We are close. Ideally happi `1.0.0` will work with pcds-devices `1.0.0`. Unfortunately, I need to wait to fix the containers until we fix the devices ...  It also seems to make sense to have the Questionnaire in there for 1.0.0 but this I'm less picky about